cns/service.go

```diff
@@ -24,11 +24,16 @@ import (
 )
 
 const (
-	// Default CNS server URL.
-	defaultAPIServerURL = "tcp://localhost:10090"
-	genericData         = "com.microsoft.azure.network.generic"
+	defaultAPIServerPort = "10090"
+	genericData          = "com.microsoft.azure.network.generic"
 )
 
+var invalidURLs = map[string]string{
+	"tcp://127.0.0.1:10090": "localhost",
+	"tcp://localhost:10090": "localhost",
+	"tcp://0.0.0.0:10090":   "default",
```

Comment:
what if its 0.0.0.0:10091? imo, should skip this invalid url check.. if user passes cns url, then we should just use it